recipes/aws-c-io/all/conanfile.py

```diff
@@ -25,22 +26,20 @@ class AwsCIO(ConanFile):
         "fPIC": True,
     }
 
-    def config_options(self):
-        if self.settings.os == "Windows":
-            del self.options.fPIC
-
-    def configure(self):
-        if self.options.shared:
-            self.options.rm_safe("fPIC")
-        self.settings.rm_safe("compiler.cppstd")
-        self.settings.rm_safe("compiler.libcxx")
+    implements = ["auto_shared_fpic"]
+    languages = "C"
 
     def layout(self):
         cmake_layout(self, src_folder="src")
 
     def requirements(self):
         # These versions come from aws-sdl-cpp prefetch_crt_dependency.sh file,
         # dont bump them independently, check the file and update all the dependencies at once
+        if self.version == "0.15.4":
+            self.requires("aws-c-common/0.11.0", transitive_headers=True, transitive_libs=True)
+            self.requires("aws-c-cal/0.8.3")
+            if self.settings.os in ["Linux", "FreeBSD", "Android"]:
+                self.requires("s2n/1.5.9")
```

Comment:
This is the only time s2n should be used in aws-c. The sdk-cpp should not, which was an issue until https://github.com/conan-io/conan-center-index/pull/26813